plugins/bingoscape-companion

```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/bingoscape/bingoscape-companion.git
+commit=7fd25ead4b469b6aed24413438c8baf5cfa56b7e
+authors=kaffeed
```

Comment:
You'll need a warning here since your domain is filled by default:
```suggestion
authors=kaffeed
warning=This plugin submits <...> and your IP to a server not controlled or verified by the RuneLite developers.
```